projects/rna/in-place-connection/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "jetpack-in-place-connection",
```

Comment:
Should this name match the name it will be published under, including the `@automattic` prefix?